scripts/collect-windows-logs.ps1

```diff
@@ -33,6 +33,22 @@ if (Test-Path $containerd) {
   }
 }
 
+$networkInterfaces = "c:\k\network-interfaces.json"
+if (Test-Path $networkInterfaces) {
+  $tempFile = Copy-Item $networkInterfaces $lockedTemp -PassThru -ErrorAction Ignore
+  if ($tempFile) {
+    $paths += $tempFile
+  }
+}
+
+$interfaces = "c:\k\interfaces.json"
+if (Test-Path $interfaces) {
+  $tempFile = Copy-Item $interfaces $lockedTemp -PassThru -ErrorAction Ignore
+  if ($tempFile) {
+    $paths += $tempFile
+  }
+}
+

```

Comment:
Could we do something like
$files = @( 
    "c:\"C:\ProgramData\containerd\root\panic.log",
     "c:\k\network-interfaces.json", 
      "c:\k\interfaces.json")

$files | Foreach-Object {
if (Test-Path $_) {
    $tempFile = Copy-Item $_ $lockedTemp -PassThru -ErrorAction Ignore
    if ($tempFile) {
      $paths += $tempFile
    }
  }
}

To reduce code duplication?

Reply:
or just add interfaces.json and network-interfaces.json to $lockedFiles above...
the panic.log was special cased b/c it was outside of c:\k

Reply:
adding the files to $lockedfiles would be the best, they will be picked up automatically

Reply:
thanks for the feedback, change made